### DISCUSSION TITLE
Java Files API cannot see files or folders of an S3 bucket mapped to a Windows drive

### DISCUSSION THREAD

**Discussion by Heliolicity** [iterate-ch/cyberduck#16499] [Q&A]
Hi all,

As the title suggests, I've mounted an S3 bucket as a Windows drive on my local machine at location D:\

I have a small Java application that I've written to scan the contents of any Windows drive or folder I specify. None of the files or folders on the mapped drive are found when using the **java.nio.Files** API. Does Mountain Duck provide support for this?

**Comment by tatze96** [comment]
Normally, this should work without issues if the API supports network drive access and the application doesn't run as a Windows service application. For more information regarding the Windows service limitation, refer to the *Windows* tab within the [Operating System Specific Issues](https://docs.cyberduck.io/mountainduck/issues/#operating-system-specific-issues) section on the Mountain Duck known issues page within our wiki.
